srcpkgs/m4ri/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'm4ri'
+pkgname=m4ri
+version=20200125
+revision=1
+build_style=gnu-configure
+configure_args="--enable-openmp"
+hostmakedepends=pkg-config
+makedepends=libpng-devel
+short_desc="Library for fast arithmetic with dense matrices over GF(2)"
+maintainer="Eloi Torrents <eloitorrentsj@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://bitbucket.org/malb/m4ri"
+distfiles="https://bitbucket.org/malb/m4ri/downloads/${pkgname}-${version}.tar.gz"
+checksum=0dfb34aed351882a0f2281535ea6f81c690a5efeb14edab131d9ba0dffe44863
+
```

Comment:
Here I suggest adding the following
```
build_options="native_build"

if [ -z "$build_option_native_build" ]; then
	# this disables sse3
	configure_args="--disable-sse2"
fi
```

If I'm not mistaken this will:
a. disable sse3 for x86_64
b. disable sse2 for i686

In particular, I expect this should fix the failure for the i686 build.

Reply:
PS: I think this is only necessary for m4ri, because m4rie will use the same cflags as m4ri.